src/precompiles/ArbWasm.sol

```diff
@@ -54,6 +54,8 @@ interface ArbWasm {
     // @return gas cost paid per half kb uncompressed.
     function callScalar() external view returns (uint16 gas);
 
+    event ProgramActivated(address program, bytes32 codehash, uint16 version);
```

Comment:
We should probably index this by codehash and program

Reply:
done.
However - two questions:
1) should we really index by program? Anyone that knows program can find codehash and index with that
2) should we rename "program" to "address"?

Reply:
I don't think we should index by program for exactly that reason.
I don't have strong feelings about renaming `program`.